src/main/java/core/basesyntax/bookstore/model/Book.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(unique = true, nullable = false)
+    private String isbn;
+
+    @Column(nullable = false)
+    private BigDecimal price;
+
+    private String description;
+    private String coverImage;
+
+    public Book(String title, String author, String isbn, BigDecimal price,
+                String description, String coverImage) {
+        this.title = title;
+        this.author = author;
+        this.isbn = isbn;
+        this.price = price;
+        this.description = description;
+        this.coverImage = coverImage;
+    }
```

Comment:
You can use Lombok's AllArgsConstructor for such trivial constructors

Reply:
it's not all args - id is excluded